.github/workflows/build-and-test.yml

```diff
@@ -22,13 +22,13 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel twine
+          pip3 install --break-system-packages setuptools wheel twine
```

Comment:
Why not use pipx and do things cleanly here?

Reply:
I have not used `pipx` a lot but I think `pipx` is not the right tool here. If I try it locally I get the following error:

```
$ pipx install setuptools

No apps associated with package setuptools or its dependencies. If you are
attempting to install a library, pipx should not be used. Consider using pip
or a similar tool instead.
```

I think the alternatives to `pip` are installation through the package manager, i.e., `apt install`, or going the `venv` path. Sure, we could do this, but I do not see a great benefit and for the sake of testing this approach is also good enough.

If you can suggest a one-liner change with `pipx`, ok. But I would rather not change more and keep it as it is. We can still change it with the next release.

Reply:
I tried to use `apt` to install the necessary build dependencies: https://github.com/precice/fenics-adapter/pull/185 but this does not seem to work. I do not want to invest too much time here, so lets go with what we have for now.